src/more-infos/more-info-media_player.html

```diff
@@ -359,6 +397,27 @@
       this.callService('select_source', { source: sourceInput });
     }
 
+    handleSoundModeChanged(soundModeIndex, soundModeIndexOld) {
+      var soundModeInput;
```

Comment:
No need to predeclare variables.

Just use `const` when you assign it.

Reply:
All handle's are done with var instead of const throught this file.
honestly I don't know the exact diffrence but I think it might be better to keep the file uniform and do the same thing as done in the rest of the file.

Basically I made a copy of the Source input, so everything is done in a similar way as the already implemented Source input.